### PR TITLE
Add OpenStack Identity Group module

### PR DESCRIPTION
**Pull request by emonty**

### CODE REVIEW

File: cloud/openstack/os_group.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+module: os_group
+short_description: Manage OpenStack Identity Groups
+extends_documentation_fragment: openstack
+version_added: "2.0"
+description:
+    - Manage OpenStack Identity Groups.
+options:
+   name:
+     description:
+        - Role Name
```
Comment (omgjlk) [inline cloud/openstack/os_group.py:36]: Should this be Group name?

File: cloud/openstack/os_group.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+     required: true
+   description:
+     description:
+        - Role Description
```
Comment (omgjlk) [inline cloud/openstack/os_group.py:40]: s/Role/Group ?

Comment (omgjlk) [inline cloud/openstack/os_group.py:40]: Docs should be updated to reflect that description can be modified on existing groups.

File: cloud/openstack/os_group.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                else:
+                    changed = False
+            module.exit_json(changed=changed, group=group)
+        elif state == 'absent':
+            if group is None:
+                changed=False
+            else:
+                cloud.delete_group(group.id)
+                changed=True
+            module.exit_json(changed=changed)
```
Comment (omgjlk) [inline cloud/openstack/os_group.py:102]: Please add check mode support.

**Comment by gregdek** [comment]
@emonty ping for update

**Comment by gregdek** [comment]
@emonty @shrews @dguerri @juliakreger @j2sol @rcarrillocruz ping, any updates here?

**Comment by gregdek** [comment]
@emonty A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

**Comment by robynbergeron** [comment]
@emonty Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If we don't hear from you within another 14 days, we will close this pull request.

**Comment by Shrews** [comment]
This has been replaced by PR: https://github.com/ansible/ansible-modules-extras/pull/1528

Please close this PR.

**Comment by bcoca** [comment]
as per above

**Pull request closed without merging**